From 7fc3fdc232cdf7c929425a2fd7bdcced160610e9 Mon Sep 17 00:00:00 2001
From: Matthias Schiffer <mschiffer@universe-factory.net>
Date: Thu, 21 Apr 2022 17:49:58 +0200
Subject: [PATCH] modules: update OpenWrt base

ecbbb373edf7 wolfssl: fix compilation with /dev/crypto
f6e22f0956a1 OpenWrt v19.07.10: revert to branch defaults
d03dc49943db OpenWrt v19.07.10: adjust config defaults
9ce6aa9d8d8a wolfssl: bump to 5.2.0
698cdf02020a mac80211: Update to version 4.19.237-1
26a8be9c9814 kernel: generic: add missing symbol for arm64 spectre mitigation
ae2af91edddd kernel: generic: reorder kernel config options
058c2347c51f imagebuilder: fix broken image generation with external targets
d39a6c67dcb4 kernel: bump 4.14 to 4.14.275
9aa35fada652 patchelf: backport fix for rpath endianness
---
 modules                                                         | 2 +-
 ...ement-MLD-Querier-wake-up-calls-Android-bug-workaround.patch | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/modules b/modules
index d068375c..abd47fc5 100644
--- a/modules
+++ b/modules
@@ -2,7 +2,7 @@ GLUON_FEEDS='packages routing gluon'
 
 OPENWRT_REPO=https://github.com/openwrt/openwrt.git
 OPENWRT_BRANCH=openwrt-19.07
-OPENWRT_COMMIT=b24905c38a8acc337a08c116b343b6ccc3ba861e
+OPENWRT_COMMIT=ecbbb373edf7be017e546be2443e6c422cb9c220
 
 PACKAGES_PACKAGES_REPO=https://github.com/openwrt/packages.git
 PACKAGES_PACKAGES_BRANCH=openwrt-19.07
diff --git a/patches/openwrt/0015-kernel-bridge-Implement-MLD-Querier-wake-up-calls-Android-bug-workaround.patch b/patches/openwrt/0015-kernel-bridge-Implement-MLD-Querier-wake-up-calls-Android-bug-workaround.patch
index d2a537d3..081c5835 100644
--- a/patches/openwrt/0015-kernel-bridge-Implement-MLD-Querier-wake-up-calls-Android-bug-workaround.patch
+++ b/patches/openwrt/0015-kernel-bridge-Implement-MLD-Querier-wake-up-calls-Android-bug-workaround.patch
@@ -1954,7 +1954,7 @@ index 0000000000000000000000000000000000000000..92bb9275df9d54778ce8f00b1cb6e999
 +2.27.0
 +
 diff --git a/target/linux/generic/config-4.14 b/target/linux/generic/config-4.14
-index 217754caa673b0226005c8e4fa7a55749ca0b6c9..d0cc204eccb9ee970790dd47aaf17d510e1374e4 100644
+index a0569e4eb0e1249880864563ee980fadcbcc4ac6..c672ee6ff8e7540920c48be9f0a45ed73e428c1b 100644
 --- a/target/linux/generic/config-4.14
 +++ b/target/linux/generic/config-4.14
 @@ -630,6 +630,7 @@ CONFIG_BRIDGE=y
-- 
2.17.1

